packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -612,6 +623,17 @@ abstract class ClusterBase extends Resource implements ICluster {
   public addHelmChart(id: string, options: HelmChartOptions): HelmChart {
     return new HelmChart(this, `chart-${id}`, { cluster: this, ...options });
   }
+
+  /**
+   * Defines a CDK8s chart in this cluster.
+   *
+   * @param id logical id of this chart.
+   * @param chart the cdk8s chart.
+   * @returns a `KubernetesManifest` construct representing the chart.
+   */
+  public addCdk8sChart(id: string, chart: cdk8s.Chart): KubernetesManifest {
+    return this.addManifest(id, ...chart.toJson());
```

Comment:
What happens with dependencies within the chart? Who takes care of them?

Reply:
`chart.toJson` does that and returns a sorted array.